src/api/.rubocop_todo.yml

```diff
@@ -1252,6 +1252,7 @@ ViewComponent/MissingPreviewFile:
     - 'app/components/delete_confirmation_dialog_component.rb'
     - 'app/components/notification_component.rb'
     - 'app/components/request_decision_component.rb'
+    - 'app/components/rpm_lint_component.rb'
```

Comment:
Any specific reason to ignore this? This is a new component, so it should be simple to fix the rubocop complaints. No?

Reply:
In this particular case we only need a preview file. I'll add it.

Reply:
Done.